packages/suite/src/hooks/suite/useFirmware.ts

```diff
@@ -15,17 +23,18 @@ export const useFirmware = () => {
         modal.windowType === 'ButtonRequest_FirmwareCheck';
 
     const actions = useActions({
```

Comment:
nit: could also get rid of `useActions`. In general, this "actions in a hook" pattern makes little sense 🤔

Reply:
I only wanted to make the smallest amount of changes as possible during this refactor so avoided this but can do this as well if you consider it critical. If not, could be handled separately later on.